rbac/policy.csv

```diff
@@ -21,6 +21,8 @@ p, admin, noderule, read, allow
 p, admin, noderule, write, allow
 p, admin, edgerule, read, allow
 p, admin, edgerule, write, allow
+p, admin, workflowcall, read, allow
+p, admin, workflowcall, write, allow
```

Comment:
workflow.call according to the suggest route

Reply:
done